paf-mvp-frontend/src/lib/paf-lib.ts

```diff
@@ -75,6 +80,32 @@ export class OneKeyLib implements IOneKeyLib {
     }
   }
 
+  parseCookieTTL(duration: string): number {
+    if (duration === undefined) {
+      this.log.Info('No cookie ttl was specified! Using default value:', DEFAULT_TTL_IN_SECONDS, 'seconds');
+      return DEFAULT_TTL_IN_SECONDS;
+    }
+    const durationInSeconds = parseDuration(duration);
+    if (durationInSeconds === null) {
+      this.log.Warn(
```

Comment:
[NIT] could be interesting to log the provided value?

Reply:
Indeed !